tests/helpers/raw-events/register-acceptance-test-helpers.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+import {
+  triggerTapHelper,
+  rawTapHelper,
+  rawPanHelper,
+  rawMouseDownHelper,
+  rawMouseUpHelper,
+  rawMouseMoveHelper,
+} from './async';
+
+export default function(){
+  Ember.Test.registerAsyncHelper('triggerTap', triggerTapHelper);
```

Comment:
import { registerAsyncHelper } from '@ember/test';

registerAsyncHelper('triggerTap', triggerTapHelper);

Reply:
This is not available in version 2.16